owlbot.py

```diff
@@ -27,52 +27,19 @@
 # ----------------------------------------------------------------------------
 # Add templated files
 # ----------------------------------------------------------------------------
-templated_files = common.py_library(
-    unit_test_python_versions=["3.6", "3.7", "3.8", "3.9"],
-    system_test_python_versions=["3.8"],
-    cov_level=50
-)
-s.move(templated_files, excludes=[
+s.move(common.py_library(), excludes=[
     # pybigquery was originally licensed MIT
-    "LICENSE"
+    "LICENSE",
+    "noxfile.py",
+    "setup.cfg",
+    ".coveragerc",
 ])
 
-# ----------------------------------------------------------------------------
-# Fixup files
-# ----------------------------------------------------------------------------
-
-s.replace(
-    ["noxfile.py"],
-    r"[\"']google[\"']",
-    '"pybigquery"',
-)
-
-s.replace(
-    ["noxfile.py"], "google/cloud", "pybigquery",
-)
-
-# Add DB config for SQLAlchemy dialect test suite.
-# https://github.com/sqlalchemy/sqlalchemy/blob/master/README.dialects.rst
-# https://github.com/googleapis/python-bigquery-sqlalchemy/issues/89
-s.replace(
-    ["setup.cfg"],
-    "universal = 1\n",
-    """universal = 1
-
-[sqla_testing]
-requirement_cls=pybigquery.requirements:Requirements
-profile_file=.profiles.txt
-
-[db]
-default=bigquery://
-bigquery=bigquery://
-"""
-)
-
```

Comment:
Is this change correct? Dropping the replacements and excluding `setup.cfg` and `noxfile.py` from synth makes them susceptible to drift.

Reply:
Done.